tests/agents/test_orchestrator.py

```
@@ -0,0 +1,366 @@
+        if hasattr(orchestrator, 'register_agent'):
+            result = await orchestrator.register_agent(mock_agent)
+            assert result is True or mock_agent.id in orchestrator.agents
+
+    @pytest.mark.asyncio
+    async def test_register_duplicate_agent_fails(self, orchestrator, mock_agent):
+        """Should reject duplicate agent registration."""
+        if hasattr(orchestrator, 'register_agent'):
+            await orchestrator.register_agent(mock_agent)
+
+            # Try to register again
+            with pytest.raises(Exception) or result:
```

Flake8 (7.3.0)

[error] undefined name 'result'

(F821)